aws/structure.go

```diff
@@ -698,15 +698,9 @@ func flattenParameters(list []*rds.Parameter) []map[string]interface{} {
 	for _, i := range list {
 		if i.ParameterName != nil {
 			r := make(map[string]interface{})
-			r["name"] = strings.ToLower(*i.ParameterName)
-			// Default empty string, guard against nil parameter values
-			r["value"] = ""
-			if i.ParameterValue != nil {
-				r["value"] = strings.ToLower(*i.ParameterValue)
-			}
-			if i.ApplyMethod != nil {
-				r["apply_method"] = strings.ToLower(*i.ApplyMethod)
-			}
+			r["name"] = aws.StringValue(i.ParameterName)
+			r["value"] = aws.StringValue(i.ParameterValue)
+			r["apply_method"] = aws.StringValue(i.ApplyMethod)
```

Comment:
This LGTM 👍 do you think we should do something similar in `func resourceAwsDbParameterHash` since that also attempts to match the result of `flattenParameters`? And then perhaps remove the StateFunc on `value` since we want to retain case?